packages/contentstack-import/src/types/import-config.ts

```diff
@@ -11,6 +11,7 @@ export interface ExternalConfig {
 }
 
 export default interface ImportConfig extends DefaultConfig, ExternalConfig {
+  importExportLogPath: string;
```

Comment:
@cs-raj it's optional param right ?

Reply:
No, as we will be assigning it some value whether explicitly passed or the backupDir path or the present working directory